src/termdict/termdict.rs

```diff
@@ -199,4 +200,13 @@ impl TermDictionary {
     pub fn stream<'a>(&'a self) -> TermStreamer<'a> {
         self.range().into_stream()
     }
+
+    pub fn search<'a, A: Automaton>(&'a self, automaton: A) -> TermStreamerBuilder<'a, A> {
+        use fst::map;
+
+        // given an Automaton and a fst::Map (fst_index)
+        // how can I generate a streambuilder?
```

Comment:
I don't understand the question here???

Reply:
oh gotcha:
``
    pub fn search<'a, A: Automaton>(&'a self, automaton: A) -> TermStreamerBuilder<'a, A> {
        let sb = self.fst_index.search(automaton);
        TermStreamerBuilder::<A>::new(self, sb)
    }`
```

Reply:
Thank you.